experimental/examples/events/README.md

```diff
@@ -0,0 +1,23 @@
+## Installation
+
+```sh
+# from this directory
+npm install
+```
+
+## Run the Application
+
+LogRecord
+
```

Comment:
```suggestion
```
Is this intended to be here? Looks like maybe got left over from something else or there is intended to be more words here, not sure.

Reply:
Oh I just realized it's on the log example too ha, probably just copied over.

Reply:
Yeah, copied. I removed it from both, not sure why it was there.